### ISSUE TITLE
Redirection error ?

### ISSUE THREAD

**Issue by cdersky** [ember-learn/guides-source#775]
1. When I go to this helpful looking guides page
[https://guides.emberjs.com/release/models/finding-records/](https://guides.emberjs.com/release/models/finding-records/) I am shown content and then redirected to an error page. Is this content still relevant? If so can it be displayed without redirecting? Example here
![gif example of redirect](http://g.recordit.co/TEl4VzgH4V.gif)

2. The error page provides a link to file issues on an [archived repo](https://github.com/ember-learn/guides-app). Can the link be updated to this repo? The copy says If you expected something else to be here, please [file a ticket](https://github.com/ember-learn/guides-app).
![gif example of link to archived repo](http://recordit.co/SFc1fB1KWX/gif/notify)

**Comment by jenweber** [comment]
Thanks for reporting the redirect issue and bad link! It seems that you are not the only person affected by this. We are debugging the redirect problem and have reverted some dependencies to see if they are the issue.

The error page link is definitely something we should fix too, once we resolve the errors themselves.

**Comment by jenweber** [comment]
Maybe addressed by #776, fingers crossed

**Comment by amyrlam** [comment]
It *seems* to be working now, but today I could only see the problem intermittently e.g. visit https://guides.emberjs.com/release/contributing/repositories/ or https://guides.emberjs.com/release/routing/query-params/, see ack page.

**Comment by amyrlam** [comment]
Spoke too soon, see ack page on a refresh 😕

**Comment by cdersky** [comment]
Thanks for the rapid response. A quick check of [https://guides.emberjs.com/release/models/finding-records/](https://guides.emberjs.com/release/models/finding-records/), [https://guides.emberjs.com/release/contributing/repositories/](https://guides.emberjs.com/release/contributing/repositories/), and [https://guides.emberjs.com/release/routing/query-params/](https://guides.emberjs.com/release/routing/query-params/) show that they seem to be working :)

**Comment by jenweber** [comment]
Closed, I haven't heard of any more problems since that one day.

**Comment by jenweber** [comment]
Just kidding, still an issue. We need to hunt this down.

**Comment by paddor** [comment]
I'm seeing this too. Here's what the console says:
```
vendor-526cb253f3a130ac1d6ecb343054adcd.js:1692 Error while processing route: version.index Ember Data Request GET /content/v3.10.0/pages.json returned a 500
Payload (text/plain; charset=utf-8)
seeker can't seek
 Error: Ember Data Request GET /content/v3.10.0/pages.json returned a 500
Payload (text/plain; charset=utf-8)
seeker can't seek
```

**Comment by mansona** [comment]
This seems to very much be an issue with our hosting provider (Netlify), I have contacted them to see what the issue is.

There are some things that we can do to potentially improve this situation and we could probably track these issues to get a sense of how many people are being affected 🤔

**Comment by bilger-progress** [comment]
Still seeing the issue...

**Comment by thomascchen** [comment]
I've been running into this issue today and yesterday with the same error as above in the console. I was actually running an Intro to Ember demo at a meetup last night, and the other folks were running into it too. Sometimes it works again when I close the browser and restart it. Reproduced this in both Firefox and Chrome.

```
Error while processing route: version.index Ember Data Request GET /content/v3.10.0/pages.json returned a 500
Payload (text/plain; charset=utf-8)
seeker can't seek
 t@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:2213:13
ue@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:6027:19
r@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:6029:56
handleResponse@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:6485:26
s@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:6450:14
ajax/</d.error/i<@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:6495:8
ajax/</d.error@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:6495:19
l@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:233:118
fireWith@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:234:694
C@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:514:468
r/<@https://guides.emberjs.com/assets/vendor-ce16731e4cf63d18470bb93b644ca744.js:524:206
```

**Comment by locks** [comment]
We are aware of the issue and it should have been addressed again!
My best explanation at the moment is that for some reason there is a Netlify node that is returning these 5xx answers when the Ember application boots up and so it transitions into an error state.
@mansona has some further mitigations planned in the event the problem reoccurs.
I'm closing this issue as we have plugged that leak for now.

Thank you everyone for participating, it has been valuable feedback!
